### PR TITLE
Update homepage styles

### PR DESCRIPTION
**Pull request by lilyndk**
## Description
Adding additional component styles to work with new content:
- Hero subtitle
- Intro "Mission" paragraph"
- "Opportunities & Programmes" section

## Change Type
- [ ] Bug Fix
- [X] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [ ] Other

## Screenshots
AFTER:
screencapture-127-0-0-1-4000-2024-04-23-21_24_46

BEFORE:
screencapture-127-0-0-1-4000-2024-04-23-21_27_12

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [X] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [X] I have tested my changes locally.
- [X] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,10 +5,40 @@
...
+
+    <div class="row justify-content-center bg-purple">
+        <div class="col-12 col-md-6 text-center pt-6 pb-4 overview">
+            <h2>Opportunities & Programmes</h2>
+            <p>Discover a world of possibilities within our community. From networking events to mentorship programs and leadership opportunities, there's something for everyone to explore. Whether you're looking to sharpen your skills, expand your professional network, or take the next step in your career, we offer a wide range of opportunities to help you achieve your goals. Join us on this journey of growth and empowerment, and unlock your full potential in the tech industry.</p>
+        </div>
+        <div class="col-12 col-md-8 text-center pb-6 programs">
+            <div class="row">
+                <div class="col">
+                    <a href="">Mentorship</a>
```
Comment (lilyndk) [inline index.html]: @dricazenck I need your help with linking these a tags to the correct program pages :)

Comment (WomenCodingCommunity) [reply index.html:23]: leave to me and I will add that! 👍

File: _sass/custom/_home.scss [hunk 2]
```diff
@@ -58,6 +59,47 @@
...
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+    }
+
+    .programs .col a:hover {
+        background-color: $primary-80;
```
Comment (dricazenck) [inline _sass/custom/_home.scss:99]: Could we break the purple with the other colour of your figma project?

File: index.html (hunk 1 above)
Comment (dricazenck) [inline index.html]:
```suggestion
                    <a href="/mentorship">Mentorship</a>
```

Comment (dricazenck) [reply index.html]: I will suggest the names you can add and I can create next the empty pages ;)

File: index.html [hunk 3]
```diff
@@ -5,10 +5,40 @@
...
+                </div>
+                <div class="col">
+                    <a href="">Online and in-person events</a>
```
Comment (dricazenck) [inline index.html]:
```suggestion
                    <a href="/events">Online and in-person events</a>
```

File: index.html [hunk 4]
```diff
@@ -5,10 +5,40 @@
...
+                </div>
+                <div class="col">
+                    <a href="">Book and write club</a>
```
Comment (dricazenck) [inline index.html]:
```suggestion
                    <a href="/book-club">Book club</a>
```

File: index.html [hunk 5]
```diff
@@ -5,10 +5,40 @@
...
+                </div>
+                <div class="w-100"></div>
+                <div class="col">
+                    <a href="">Resume clinic</a>
```
Comment (dricazenck) [inline index.html]:
```suggestion
                    <a href="/interview-preparation#resume-clinic">Resume clinic</a>
```

File: index.html [hunk 6]
```diff
@@ -5,10 +5,40 @@
...
+                </div>
+                <div class="col">
+                    <a href="">Mock interviews</a>
```
Comment (dricazenck) [inline index.html]:
```suggestion
                    <a href="/interview-preparation#mock-interview">Mock interviews</a>
```

File: index.html [hunk 7]
```diff
@@ -5,10 +5,40 @@
...
+                </div>
+                <div class="col">
+                    <a href="">Leetcode</a>
```
Comment (womencodingcommunity) [inline index.html]:
```suggestion
                    <a href="/interview-preparation#leetcode">Leetcode</a>
```

**Review by dricazenck — APPROVED** [review]

File: _sass/custom/_home.scss (hunk 2 above)
Comment (lilyndk) [reply _sass/custom/_home.scss:99]: As discussed with Elena in Slack, we should vote on the secondary colour later and only use the primary tones for now :)

**Pull request merged**